src/Application.php

```diff
@@ -111,96 +128,111 @@ public function getRoute()
         return new $route_class;
     }
 
-    /**
-     * @return Config
-     */
-    public function getConfig()
+    public function getConfig(): ?Config
     {
         return $this->config;
     }
 
-    public function setDir($dirname, $path)
+    public function setDir($dirname, $path): void
     {
         $this->dirs[$dirname] = $path;
     }
 
+    /**
+     * @return mixed
```

Comment:
I don't this this `mixed` is useful.
It's not clear whether you want to keep it "mixed" permanently, or  just you want explain "what is the current behavior".
As for the latter, "mixed" is obvious.